### DISCUSSION TITLE
about Modelica.Fluid.Vessels.ClosedVolume

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10613] [Q&A] [answered]
Hello!

I am a beginner, and I'm wondering whether Modelica.Fluid.Vessels.ClosedVolume in OMEdit distinguishes between inlet and outlet, even though when connected, the lines appear to be connected at the same location?

**Comment by casella** [comment, marked as answer]
The closed volume has an array of ports, every time you connect to it the size of the array is automatically increased. They all support flow reversal, so there is no difference between inlet and outlet.

**Reply by ZackeyZAKI** [reply]
Thank you for answering!
